client/src/main/java/org/evosuite/TestSuiteGenerator.java

```diff
@@ -82,766 +87,766 @@
-	 * @return null if there were no thrown instances of ExceptionInInitializerError
-	 */
-	private static ExceptionInInitializerError getInitializerError(ExecutionResult execResult) {
-		for (Throwable t : execResult.getAllThrownExceptions()) {
-			if (t instanceof ExceptionInInitializerError) {
-				ExceptionInInitializerError exceptionInInitializerError = (ExceptionInInitializerError)t;
-				return exceptionInInitializerError;
-			}
-		}
-		return null;
-	}
-
-	/**
-	 * Reports the initialized classes during class initialization to the
-	 * ClassReInitializater and configures the ClassReInitializer accordingly
-	 */
-	private void configureClassReInitializer() {
-		// add loaded classes during building of dependency graph
-		ExecutionTrace execTrace = ExecutionTracer.getExecutionTracer().getTrace();
-		final List<String> initializedClasses = execTrace.getInitializedClasses();
-		ClassReInitializer.getInstance().addInitializedClasses(initializedClasses);
-		// set the behaviour of the ClassReInitializer
-		final boolean reset_all_classes = Properties.RESET_ALL_CLASSES_DURING_TEST_GENERATION;
-		ClassReInitializer.getInstance().setReInitializeAllClasses(reset_all_classes);
-	}
-
-	private static void writeJUnitTestSuiteForFailedInitialization() throws EvosuiteError {
-		TestSuiteChromosome suite = new TestSuiteChromosome();
-		DefaultTestCase test = buildLoadTargetClassTestCase(Properties.TARGET_CLASS);
-		suite.addTest(test);
-		writeJUnitTestsAndCreateResult(suite);
-	}
-
-	/**
-	 * Creates a single Test Case that only loads the target class. 
-	 * <code>
-	 * Thread currentThread = Thread.currentThread();
-	 * ClassLoader classLoader = currentThread.getClassLoader();
-	 * classLoader.load(className);
-	 * </code>
-	 * @param className the class to be loaded
-	 * @return
-	 * @throws EvosuiteError if a reflection error happens while creating the test case
-	 */
-	private static DefaultTestCase buildLoadTargetClassTestCase(String className) throws EvosuiteError {
-		DefaultTestCase test = new DefaultTestCase();
-
-		StringPrimitiveStatement stmt0 = new StringPrimitiveStatement(test, className);
-		VariableReference string0 = test.addStatement(stmt0);
-		try {
-			Method currentThreadMethod = Thread.class.getMethod("currentThread");
-			Statement currentThreadStmt = new MethodStatement(test,
-					new GenericMethod(currentThreadMethod, currentThreadMethod.getDeclaringClass()), null,
-					Collections.emptyList());
-			VariableReference currentThreadVar = test.addStatement(currentThreadStmt);
-
-			Method getContextClassLoaderMethod = Thread.class.getMethod("getContextClassLoader");
-			Statement getContextClassLoaderStmt = new MethodStatement(test,
-					new GenericMethod(getContextClassLoaderMethod, getContextClassLoaderMethod.getDeclaringClass()),
-					currentThreadVar, Collections.emptyList());
-			VariableReference contextClassLoaderVar = test.addStatement(getContextClassLoaderStmt);
+    private static final String FOR_NAME = "forName";
```

Comment:
It's difficult to see what has changed since this change includes a reformatting. Is the reformatting on purpose, or can we reduce the change to the actual changes?

Reply:
No, the reformatting was not on purpose, i will undo the reformatting and commit with only the actual changes.